modules/openy_features/openy_prgf/modules/openy_prgf_featured_blogs/config/install/field.field.paragraph.featured_blogs.field_fblog_posts.yml

```diff
@@ -10,7 +10,7 @@ field_name: field_fblog_posts
 entity_type: paragraph
 bundle: featured_blogs
 label: 'Blog Posts'
-description: ''
+description: 'Multiple values. Reference field to Blog posts.�'
```

Comment:
Weird symbol at the end.
![image](https://user-images.githubusercontent.com/1316234/27126699-06f51720-5101-11e7-8ca9-ece7f51782f0.png)